src/components/toast/toast.stories.mdx

```diff
@@ -63,7 +63,7 @@ Please note, by clicking a `toggle preview` button it will open example toast at
 <Preview>
   <Story
     name="variant-info"
-    parameters={{ info: { disable: true }}}
+    parameters={{ info: { disable: true }, chromatic: { disable: true }}}
```

Comment:
we can put `parameters={{ info: { disable: true }, chromatic: { disable: true }}}` into a `<Meta>` tag and ignore all of the stories in this mdx instead of putting it in every `<Story>` separately